server/valgrind-out.txt

```diff
@@ -1,131 +1,152 @@
+--869089-- REDIR: 0x4912390 (libc.so.6:strcpy) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--869089-- REDIR: 0x4913b10 (libc.so.6:strcasecmp_l) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--869089-- REDIR: 0x49126a0 (libc.so.6:strlen) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--869089-- REDIR: 0x49128d0 (libc.so.6:strncpy) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--869089-- REDIR: 0x4913950 (libc.so.6:stpcpy) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--869089-- REDIR: 0x49133f0 (libc.so.6:memchr) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--869089-- REDIR: 0x4912b00 (libc.so.6:strspn) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--869089-- REDIR: 0x49137d0 (libc.so.6:mempcpy) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--869089-- REDIR: 0x4913ac0 (libc.so.6:strncasecmp) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--869089-- REDIR: 0x4914f80 (libc.so.6:rawmemchr) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--869089-- REDIR: 0x4a07610 (libc.so.6:__strrchr_avx2) redirected to 0x484e810 (rindex)
+--869089-- REDIR: 0x4a0af80 (libc.so.6:__memset_avx2_unaligned_erms) redirected to 0x4852770 (memset)
+--869089-- REDIR: 0x490f0a0 (libc.so.6:malloc) redirected to 0x4848820 (malloc)
+--869089-- REDIR: 0x490f3e0 (libc.so.6:free) redirected to 0x484b210 (free)
+--869089-- REDIR: 0x4910520 (libc.so.6:calloc) redirected to 0x484d9d0 (calloc)
+--869089-- REDIR: 0x4a077e0 (libc.so.6:__strlen_avx2) redirected to 0x484ed10 (strlen)
+--869089-- REDIR: 0x4a0a7c0 (libc.so.6:__memcpy_avx_unaligned_erms) redirected to 0x4852880 (memmove)
+--869089-- REDIR: 0x4912fc0 (libc.so.6:__GI_strstr) redirected to 0x4853ae0 (__strstr_sse2)
+--869089-- REDIR: 0x4a03a80 (libc.so.6:__memcmp_avx2_movbe) redirected to 0x4852010 (bcmp)
+--869089-- REDIR: 0x4a02940 (libc.so.6:__strcmp_avx2) redirected to 0x484fbd0 (strcmp)
+--869089-- REDIR: 0x4a07400 (libc.so.6:__strchrnul_avx2) redirected to 0x4853330 (strchrnul)
+--869089-- REDIR: 0x4a0a780 (libc.so.6:__mempcpy_avx_unaligned_erms) redirected to 0x4853440 (mempcpy)
+--869089-- REDIR: 0x490f740 (libc.so.6:realloc) redirected to 0x484dc50 (realloc)
+--869089-- REDIR: 0x4a08cb0 (libc.so.6:__strcpy_avx2) redirected to 0x484ee60 (strcpy)
+--869089-- REDIR: 0x4a07180 (libc.so.6:__strchr_avx2) redirected to 0x484e9f0 (index)
+--869089-- REDIR: 0x4a099a0 (libc.so.6:__stpcpy_avx2) redirected to 0x4852130 (stpcpy)
+==869089== 
+==869089== HEAP SUMMARY:
+==869089==     in use at exit: 408 bytes in 2 blocks
+==869089==   total heap usage: 267 allocs, 265 frees, 540,411 bytes allocated
+==869089== 
+==869089== Searching for pointers to 2 not-freed blocks
+==869089== Checked 8,493,576 bytes
+==869089== 
+==869089== 136 bytes in 1 blocks are still reachable in loss record 1 of 2
+==869089==    at 0x4848899: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
+==869089==    by 0x10B47E: main (in /home/student/Documents/AESD/assignments-3-and-later-shruthi-thallapally/server/aesdsocket)
```

Comment:
@shruthi-thallapally this is probably a real issue, compile with `-g` and you should get line numbers.